### ISSUE TITLE
Usb device redirecting not working

### ISSUE THREAD

**Issue by dd-jacks** [FreeRDP/FreeRDP#2113]
Currently pulled.

In associate with issue #2078. New log messages not showing fatal handler: signum=11, but it's still crashing.
command line:

```
user@ubuntu-test:~/FreeRDP-master$ sudo xfreerdp /u:"user" /p:"password" /v:XXX.XXX.XXX.XXX /f /rfx /usb:id,dev:04b8:014a
```

log is:

```
[10:06:54:033] [6578:6579] [INFO][com.freerdp.client.common.cmdline] - loading channel drdynvc
[10:06:54:086] [6578:6579] [ERROR][com.freerdp.locale] - Unable to get current timezone rule
[10:06:55:215] [6578:6580] [INFO][com.freerdp.channels.dvcman.client] - Loading Dynamic Virtual Channel urbdrc
[10:06:55:224] [6578:6580] [WARN][com.freerdp.channels.urbdrc.client] - bus:0 dev:0 not exist in udevman
[10:06:55:224] [6578:6580] [ERROR][com.freerdp.channels.urbdrc.client] - VID: 0x04B8 PID: 0x014A
[10:06:55:229] [6578:6580] [WARN][com.freerdp.channels.urbdrc.client] - bus:8 dev:3 not exist in udevman
```

It gave some new information about the problem that udevman is guilty

**Comment by HenryJacques** [comment]
@dd-jacks what do you mean by "device redirected successfully"?
Can you *really* use it in the remote session?
(see #2832)

**Comment by dd-jacks** [comment]
@HenryJacques 
Yes, I successfully use my web-camera!
I've got some troubles with epson scanner, but I have the same troubles with mstsc.

**Comment by HenryJacques** [comment]
@dd-jacks 
Thanks for your feedback. Can you give me your exact version of xfreerdp? (commit)

**Comment by dd-jacks** [comment]
e45ebd8

**Comment by HenryJacques** [comment]
Great thanks!

**Comment by lumm** [comment]
I have tried to use root user that can solve this problem.

**Comment by akallabeth** [comment]
Fixed with #5745
